server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryLifecycle.java

```diff
@@ -64,8 +86,13 @@ public void transform() {
         myApp = visitor.getFilledApp();
 
         try {
-            CloudFoundryFileCreator fileCreator = new CloudFoundryFileCreator(context.getPluginFileAccess(), myApp);
+            CloudFoundryFileCreator fileCreator = new CloudFoundryFileCreator(fileAccess, myApp);
             fileCreator.createFiles();
+            if (cloudFoundryConnection != null) {
+                CloudFoundryInjectionHandler injectionHandler = new CloudFoundryInjectionHandler(fileAccess, myApp);
+                injectionHandler.injectServiceCredentials();
+                fileCreator.updateManifest();
+            }
         } catch (IOException e) {
```

Comment:
Does this exception and the next need different treatment? It not, please combine them using `catch (IOException | JSONException e)` (the syntax might differ)